### DISCUSSION TITLE
An achievement for reaching some Followers count

### DISCUSSION THREAD

**Discussion by Peter-developer01** [community/community#61281] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Maybe add an achievement for reaching some followers count? For example, as the Starstruck badge.

**Comment by GhostOf0days** [comment]
That just doesn't make sense. Followers are meant to validate someone as being a trusted developer or leader. An achievement rewarding reaching some follower count encourages some people to use bot accounts, etc. to raise their follower count. This then defeats the purpose of followers.
